config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
But I'm using it intentionally for the scenario, so I can reference it this way:
https://github.com/jsuchome/cct/blob/glance_to_master/features/step_definitions/glance/image_create_steps.rb#L4